src/app/admin/_components/faq/hooks/useActiveFaqs.js

```diff
@@ -0,0 +1,18 @@
+import { useGetList } from 'react-admin';
+
+export const useActiveFaqs = ({ page, perPage } = { page: 1, perPage: 50 }) => {
```

Comment:
what do you think about generalizing this hook and passing isActive as an argument of the hook?

Maybe even if it is possible you can pass this argument on to useGetList hook.

Reply:
1. we can pass `isActive` but firstly, its the only case in app, and secondly, im afraid that would bring more questions, like `why do we not pass any filtering args` :) 
2. it basically to work as alias to `useGetMany` hook, to simplify usage